src/handleApps.test.ts

```diff
@@ -174,6 +175,49 @@ describe(`handleApps`, () => {
       "public, max-age=31536000, immutable"
     );
   });
+
+  it(`can retrieve a file from s3`, async () => {
```

Comment:
Please add one more test that verifies the correct 404 response when s3 responds with a not found error. You'd need to mock s3 for it to throw an error

```js
sendMock.mockImplementationOnce(() => {
  const err = Error();
  err.Code = "NoSuchKey";
  throw err;
});
```